### ISSUE TITLE
No mention of the -RestoreTargetFolder parameter

### ISSUE THREAD

**Issue by JeremyTBradshaw** [MicrosoftDocs/office-docs-powershell#10283]
I would submit a PR but in this case I don't want to get anything wrong since I'm not in the insider's circle there.  Please add this parameter to the doc.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 21b1cd9d-df2e-d4d6-52e2-d3fbdfb9db9f
* Version Independent ID: e675173d-290a-dee0-62a4-5485cf3d2570
* Content: [Restore-RecoverableItems (ExchangePowerShell)](https://learn.microsoft.com/en-us/powershell/module/exchange/restore-recoverableitems?view=exchange-ps)
* Content Source: [exchange/exchange-ps/exchange/Restore-RecoverableItems.md](https://github.com/MicrosoftDocs/office-docs-powershell/blob/main/exchange/exchange-ps/exchange/Restore-RecoverableItems.md)
* Service: **exchange-powershell**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by chrisda** [comment]
@JeremyTBradshaw, the parameter has been added to the topic per the referenced pull request. Feel free to add a description using the GitHub backend version of the topic (updated topic likely won't publish to docs.microsoft.com until Tu 1/3 earliest):

<https://github.com/MicrosoftDocs/office-docs-powershell/blob/main/exchange/exchange-ps/exchange/Restore-RecoverableItems.md>

Instructions for how to edit are [here](https://github.com/MicrosoftDocs/office-docs-powershell/blob/main/README.md). Be sure to @ include me in your pull request.

Also, I discovered another parameter named PolicyTag, which I also added. Feel free to add a description for that one, too if you know what it's for.
